swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md

```diff
@@ -68,6 +68,10 @@ You can publish a Ballerina archive to the <a href="https://central.ballerina.io
 
 3. Download and place the `Settings.toml` file in your home repository (`<USER_HOME>/.ballerina/`). If you already have a `Settings.toml` file configured in your home repository, follow the other option and copy the access token into the `Settings.toml`. 
 
+Note :  These generated tokens are valid only for an year from the generation. Upon the expiration of
+the token, a new token should be generated and `Settings.toml` should be updated with it. Otherwise, you
+may get any error related to unauthorised access.
```

Comment:
```suggestion
may get an unauthorized access error.
```

Reply:
My suggestion : 

Note: These generated tokens are valid only for one year from the generation date. Upon the expiration of the token, a new token should be generated, and Settings.toml should be updated with it. Otherwise, you may get an error related to unauthorised access.

Reply:
Still not addressed?

Reply:
Addressed now